module/TaskManagement/src/TaskManagement/Controller/RemindersController.php

```diff
@@ -0,0 +1,113 @@
+<?php 
+
+namespace TaskManagement\Controller;
+
+use ZFX\Rest\Controller\HATEOASRestfulController;
+use TaskManagement\Service\NotifyMailListener;
+use TaskManagement\Service\TaskService;
+use TaskManagement\Entity\Task;
+use Zend\Permissions\Acl\Acl;
+use TaskManagement;
+use TaskManagement\Entity\TaskMember;
+use Zend\View\Model\ViewModel;
+
+
+
+
+use Zend\View\Renderer\PhpRenderer;
+use Zend\View\Resolver\TemplateMapResolver;
+
+class RemindersController extends HATEOASRestfulController
+{
+
+	protected static $collectionOptions = ['POST'];
+	protected static $resourceOptions = [];
+	
+	/**
+	 *
+	 * @var NotifyMailListener
+	 */
+	protected $notifyMailListener;
+	/**
+	 *
+	 * @var TaskService
+	 */
+	protected $taskService;
+	/**
+	 *
+	 * @var \DateInterval
+	 */
+	protected $intervalForRemindAssignmentOfShares;
+
+	/**
+	 *
+	 * @var Acl
+	 */
+	private $acl;
+	
+ 	public function __construct(NotifyMailListener $notifyMailListener, TaskService $taskService, Acl $acl) {
+ 		
+ 		$this->notifyMailListener = $notifyMailListener;
+ 		$this->taskService = $taskService;
+ 		$this->acl = $acl;
+ 		$this->intervalForRemindAssignmentOfShares = self::getDefaultIntervalToRemindAssignmentOfShares();
+ 	}
+	
+	/**
+	 * Create a new reminder
+	 * @method POST
+	 * @link http://oraproject/task-management/tasks/reminders/
+	 * @param array $data['reminder'] 
+	 * @return HTTPStatusCode
+	 * 
+	 */
+	public function create($data){
+		
+		if(!$this->acl->isAllowed($this->identity()['user'], NULL, 'TaskManagement.Reminder.createReminder')){
```

Comment:
la funzione isAllowed è disponibile attraverso un plugin del controller, quindi non devi passare attraverso il servizio $acl, puoi chiamare direttamente `$this->isAllowed()`. La dipendenza dal servizio $acl va quindi rimossa